src/main/site/assets/less/layout.less

```diff
@@ -30,10 +30,6 @@ body {
   padding: 0;
 }
 
-.holder {
```

Comment:
Where was it used? The class is removed from the css, but no classes are removed from the html/md files?